src/XrdSecgsi/XrdSecgsitest.cc

```diff
@@ -87,7 +87,11 @@ static void pdots(const char *t, bool ok = 1)
    unsigned int i = 0;
    unsigned int l = (t) ? strlen (t) : 0;
    unsigned int np = PRTWIDTH - l - 8;
-   printf("|| %s ", t);
+   if (l > 0) {
+      printf("|| %s ", t);
+   } else {
+      printf("||  ");
+   }
```

Comment:
How about
```suggestion
   printf("|| %s ", t ? t : "");
```

Reply:
I based the proposed change to the style used a few lines further down in the file to be consistent:

https://github.com/xrootd/xrootd/blob/333a5dd8187a897bc2c3c5e8097be0aef11b8e7e/src/XrdSecgsi/XrdSecgsitest.cc#L99-L103

Reply:
Ok, let's keep it this way then.